perception/image_projection_based_fusion/src/pointpainting_fusion/node.cpp

```diff
@@ -133,7 +133,7 @@ PointPaintingFusionNode::PointPaintingFusionNode(const rclcpp::NodeOptions & opt
     auto it = find(paint_class_names.begin(), paint_class_names.end(), cls);
     if (it != paint_class_names.end()) {
       int index = it - paint_class_names.begin();
-      class_index_[cls] = index + 1;
+      class_index_[cls] = pow(2, index);
```

Comment:
The code is tricky and difficult to understand. Could you please comment or refactor it?

Reply:
I added in description and comment https://github.com/autowarefoundation/autoware.universe/pull/4705/commits/b730594e289fccfab5936eb5547faeee4dfee7e7#diff-e8df78707e8cee5e869c96c349aca84fa2126bc70e25f093bf10f8a325e638e2R136